instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/test/opmsg_deployment.go

```diff
@@ -0,0 +1,237 @@
+// Copied from https://github.com/mongodb/mongo-go-driver/blob/d46f29c34e9ff3968a5c00dcdea8fb41a20a2b2f/mongo/integration/mtest/opmsg_deployment.go
```

Comment:
This file was copied to avoid taking a dependency on the `mtest` package, which is being removed in 2.0.  See [GODRIVER-2724](https://jira.mongodb.org/browse/GODRIVER-2724) fore more information.

Reply:
Here is a blog post describing how to use`mtest` to mock, if the team would rather go this way: https://medium.com/@victor.neuret/mocking-the-official-mongo-golang-driver-5aad5b226a78